src/Handler/Search.hs

```diff
@@ -59,7 +59,7 @@ routeResult SearchResult{..} =
       ( PackageVersionModuleDocsR ppkgName pversion modName
       , Nothing
       )
-    DeclarationResult typeOrValue modName declTitle typeText ->
+    DeclarationResult typeOrValue modName declTitle _typeText ->
```

Comment:
Is this change because it was generating an unused binding warning? If that's the case I think I'd rather just use `_`, since I don't think the leading underscore approach is very well known (in fact, I had to go and look it up just now).